### ISSUE TITLE
Local ARA testing after Canary update to 113

### ISSUE THREAD

**Issue by dmdabbs** [GoogleChromeLabs/privacy-sandbox-dev-support#88]
ARA is no longer available to test (attributionsrc no longer firing) after Canary updated to 113. 

The _Privacy Sandbox Ads APIs_ flag is unavailable. 
I enabled the new _Privacy Sandbox Settings V4_ flag. In the new UX, the 

     Ad measurement
     Sites and advertisers can measure the performance of their ads

feature, that I assume drives ARA, is toggled on. But no go. 

I can retool and install Beta, but would prefer to continue using canary.

**Comment by johnivdel** [comment]
Thanks for filing this! This flag expired unintentionally and the fix is being tracked here: https://bugs.chromium.org/p/chromium/issues/detail?id=1420018

Unfortunately there is not easy way to get the flag back until the fix lands. Until then you could either use Chrome beta or,
start Chrome Canary with the following additional command line argument:

--enable-privacy-sandbox-ads-apis

**Comment by dmdabbs** [comment]
Yes, I just found the patch while in Monorail prior to seing your response here.
Thank you!

**Comment by dmdabbs** [comment]
In the meantime I'll install Beta (should have it anyway).

@johnivdel, I'm curious if you have a sense for how long the patch would land in Canary?

One other thing. Once it lands, I'd like to check that my OT token(s) will workas expected. Is the only way to see that to repeatedly reinstall Chrome till one is included in the experiment group?

**Comment by miketaylr** [comment]
Assuming that patch lands today, it should be in Canary tomorrow (if not later today).

Another thing you can do is to enable chrome://flags/#temporary-unexpire-flags-m112, and after you restart you'll have the flag available. Should tide you over until the fix lands.

**Comment by dmdabbs** [comment]
Testing with Beta I no longer see Attribution-Success Debug Reports in chrome://attribution-internals/. 
Was this a recent vintage feature only available in Chrome, or have I somehow fubared my tests?

**Comment by Mariamyoussef192** [comment]
What can I do
